### PR TITLE
fix: List V9 - update stories to work with dark themes

### LINKED ISSUE #32366: Docs: List V9 dark/HC theme in docs not working properly
Currently the examples in the docs for Fluent V9 List don't work properly on dark and HC themes. This should be fixed before going to stable.

### PR DESCRIPTION
**Pull request by george-cz**
fixes #32366

### CODE REVIEW

File: packages/react-components/react-list-preview/library/src/components/ListItem/useListItem.tsx [hunk 1]
```diff
@@ -175,7 +176,7 @@ export const useListItem_unstable = (
   });
 
   const tabsterAttributes = useMergedTabsterAttributes_unstable(
-    focusableItems ? arrowNavigationAttributes : {},
```
Comment (george-cz) [inline packages/react-components/react-list-preview/library/src/components/ListItem/useListItem.tsx:178]: this was breaking build

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-list-preview/library/src/components/ListItem/useListItem.tsx (hunk 1 above)
Comment (bsunderhus) [reply packages/react-components/react-list-preview/library/src/components/ListItem/useListItem.tsx:178]: This was modified here: https://github.com/microsoft/fluentui/pull/32300/files#diff-d3287bdc67e7afdeb4149b129ca698a5a9eb873eebee2cb3fb998a99d7e46a79

I'm a little surprised with this 👀👀, I'll investigate

Comment (george-cz) [reply packages/react-components/react-list-preview/library/src/components/ListItem/useListItem.tsx]: resolved, this was my dependencies not building properly :)

**Review by bsunderhus — APPROVED** [review]

**Pull request merged**